releases/4.0.2.md

```diff
@@ -0,0 +1,194 @@
+category: Releases
+---
+
+# TiDB 4.0.2 Release Notes
+
+Release date: Month Date, Year
+
+TiDB version: 4.0.2
+
+## Compatibility Changes
+
++ TiDB
+    - Remove sensitive information in slow query log and statement summary table. [#18130](https://github.com/pingcap/tidb/pull/18130)
+
++ TiKV
+    -
++ PD
+    -
++ TiFlash
+    - Improve backward compatibility when upgrading from the older version. [#786](https://github.com/pingcap/tics/pull/786)
++ Tools
+    - TiDB Binlog
+        -
+    - TiCDC
+        -
+    - Lignting
+        -
+
+## New Features
+
++ TiDB
+    - Add new aggregate function `APPROX_COUNT_DISTINCT()` to calculate the approximate result of `COUNT(DISTINCT)`. [#18120](https://github.com/pingcap/tidb/pull/18120)
+    - Support `MEMORY_QUOTA()` hint in `INSERT` statements. [#18101](https://github.com/pingcap/tidb/pull/18101)
+    - Add config `enable-collect-execution-info` and session variable `tidb_enable_collect_execution_info` to control whether to collect execution information of each operator and record it in the slow query log. [#18073](https://github.com/pingcap/tidb/pull/18073), [#18072](https://github.com/pingcap/tidb/pull/18072)
+    - Support authentication based on TLS certificate SAN field [#17698](https://github.com/pingcap/tidb/pull/17698)
+    - Add the `STATUS_ADDRESS` column in the `INFORMATION_SCHEMA.INSPECTION_RESULT` table to indicate the status address of servers. [#17695](https://github.com/pingcap/tidb/pull/17695)
+    - Add the `SOURCE` column in the `MYSQL.BIND_INFO` table to indicate the how the bindings are created. [#17587](https://github.com/pingcap/tidb/pull/17587)
+    - Add the global variable `tidb_slow_log_masking` to control whether to desensitize the query in slow query log. [#17694](https://github.com/pingcap/tidb/pull/17694)
+    - Support spliting the region for the newly added partition by default. [#17665](https://github.com/pingcap/tidb/pull/17665)
+    - Enable `IF()/BITXOR()/BITNEG()` to be pushed down to TiFlash Coprocessor. [#17651](https://github.com/pingcap/tidb/pull/17651)
+    - Support the `sql_select_limit` session and global vairable. [#17604](https://github.com/pingcap/tidb/pull/17604)
+    - Support collation for the `REGEXP()` function. [#17581](https://github.com/pingcap/tidb/pull/17581)
+    - Enable `JSON_LENGTH()` to be pushed down to TiFlash Coprocessor. [#17592](https://github.com/pingcap/tidb/pull/17592)
+
++ TiKV
+    -
++ PD
+    -
++ TiFlash
+    - Support new aggregation function APPROX_COUNT_DISTINCT. [#798](https://github.com/pingcap/tics/pull/798)
+    - Enable rough set filter feature by default. [#777](https://github.com/pingcap/tics/pull/777)
+    - Enable TiFlash running on ARM architecture. [#769](https://github.com/pingcap/tics/pull/769)
+    - Reduce memory consumption of delta index. [#787](https://github.com/pingcap/tics/pull/787)
+    - Better update algorithm of delta index. [#794](https://github.com/pingcap/tics/pull/794)
```

Comment:
Better change to something like: "Use more efficient update algorithm for delta index".